src/components/LessonActions/index.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react'
+import LessonStateProgression from './components/LessonStateProgression'
+import LessonEdit from './components/LessonEdit'
+
+export default ({lesson}) => (
+  <div>
+    <LessonEdit lesson={lesson} />
+    <LessonStateProgression lesson={lesson} />
```

Comment:
This top level `LessonActions` component brings together all the actions you can take on a lesson based on the hypermedia available:

<img width="466" alt="screen shot 2017-03-07 at 10 16 55 pm" src="https://cloud.githubusercontent.com/assets/5497885/23691502/1218bbb6-0386-11e7-9554-287c86811ba0.png">